task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -86,3 +91,4 @@ spec:
           HACBS_TEST_OUTPUT="$(make_result_json -r SUCCESS -s $check_num)"
           echo "${HACBS_TEST_OUTPUT}" | tee $(results.HACBS_TEST_OUTPUT.path)
         fi
+        cd .. && rm -fr to-delete
```

Comment:
bonus pints for making `to-delete` a constant instead of copy-pasting the same name 3 times